ide/db.drivers/src/org/netbeans/modules/db/drivers/PostgreSQLDriver.xml

```diff
@@ -39,7 +39,7 @@
         <!-- please check with mkleint@netbeans.org before/after updating this or "classpath" section -->
         <property>
             <name>maven-dependencies</name>
-            <value>org.postgresql:postgresql:9.4.1209:jar</value>
+            <value>org.postgresql:postgresql-42.2.5.jar:jar</value>
```

Comment:
This should be: org.postgresql:postgresql:42.2.5:jar (the ".jar" in the middle does not belong there). This are maven coordinates in the compact form.

Reply:
Fixed in 4e44c52268902c1213110ad0eb33863c2cba7e0f.